src/lib/combat.ts

```diff
@@ -54,7 +74,8 @@ export class Macro extends LibramMacro {
         .trySkill($skill`Extract`)
         .tryItem($item`porquoise-handled sixgun`)
         .trySkill($skill`Bowl a Curveball`)
-        .trySkill($skill`Asdon Martin: Spring-Loaded Front Bumper`),
+        .trySkill($skill`Asdon Martin: Spring-Loaded Front Bumper`)
+        .tryCLEESH(),
```

Comment:
This isn't a free run anymore; let's break out another method that pulls in the static `freeRun` macro, and appends cleesh.